app/src/main/java/com/framgia/forder/data/source/remote/api/response/UpdateProfileResponse.java

```diff
@@ -1,65 +1,21 @@
 package com.framgia.forder.data.source.remote.api.response;
 
-import android.os.Parcel;
-import android.os.Parcelable;
-import com.framgia.forder.data.source.remote.api.request.UpdateProfileRequest;
+import com.framgia.forder.data.model.User;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.model.User. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports